live-editing/configs/TreeGridConfigGenerator.ts

```diff
@@ -565,6 +566,18 @@ export class TreeGridConfigGenerator implements IConfigGenerator {
             shortenComponentPathBy: "/tree-grid/"
         }));
 
+        // TreeGrid Multi Column Header Template sample
+        configs.push(new Config({
+            additionalFiles: ["/src/app/tree-grid/data/employees-flat-detailed.ts"],
+            appModuleConfig: new AppModuleConfig({
+                imports: [IgxTreeGridModule, TreeGridMultiColumnHeaderTemplateSampleComponent, IgxButtonModule],
```

Comment:
IgxButtonModule does not need to be imported separately (imports and ngImports).

Reply:
Done.